src/main/resources/StageSpecifics_LOC.properties

```diff
@@ -26,6 +26,8 @@ scenariosroot4.description=Gerrits workspace src/test/resources/TestTool
 scenariosroot5.directory=C:/Gerrit/Workspaces/eclipse/${instance.name.lc}/src/test/testtool
 scenariosroot5.description=Gerrits workspace src/test/testtool
 
+scenariosroot6.directory=C:/Users/martijn/PerformanceTest/ibis4pt/src/test/testtool
+scenariosroot6.description=Martijn hard-code path
```

Comment:
Kan je proberen een specificatie te maken zonder harde paden (zonder 'Martijn' of 'Gerrit', zeg maar)?

Reply:
Ik heb moeite gehad om ibis4pt aan de gang te krijgen. Met deze fix is het gelukt. Ik zou liever geen tijd besteden aan een betere oplossing. Wil je echt dat ik hier tijd in stop?

Reply:
OK, laat voorlopig maar even